app/hallebarde/infrastructure/token_repository.py

```diff
@@ -0,0 +1,14 @@
+import boto3
+import hallebarde.config
+
+client = boto3.resource('dynamodb')
+table = client.Table(f'hallebarde-{hallebarde.config.ENVIRONMENT}-table')
+
+
+def save(id: str, upload_token: str, download_token: str):
+    table.put_item(Item={'id': id, 'upload_token': upload_token, 'download_token': download_token})
+
+
+def get(id: str):
```

Comment:
I would add some type hinting about the return value (is it a dict or something ? Maybe we can model a dedicated object, like a Token or something)

Reply:
Yes we shall. I will try to implement that.